addons/mail/tools/link_preview.py

```diff
@@ -28,6 +33,12 @@ def get_link_preview_from_url(url, request_session=None):
             response = requests.get(url, timeout=3, headers=user_agent, allow_redirects=True, stream=True)
     except requests.exceptions.RequestException:
         return False
+    except LocationParseError as e:
+        _logger.warning(
+            "The provided url is problematic: %s, the error was risen: %s",
+            url, str(e)
+        )
```

Comment:
Hello, thank you for the PR.
Is anyone going to look at these warning?
It's probably best to just return False, like for other errors.

Reply:
@seb-odoo no, but i think it's still helpful, no? i personally sometimes look at logs, but if u insist i can remove it

Reply:
Helpful for who? A developer debugging maybe yes.

But logs are also and mainly made for sys admin running Odoo in production, and the invalid URL is user input, there is no action the sys admin can do to fix it!

Reply:
@seb-odoo done